tests/unit/backend/wmg/fixtures/test_snapshot.py

```diff
@@ -277,6 +305,32 @@ def create_expression_summary_cube(
     return cube_dir
 
 
+def create_expression_summary_fmg_cube(
+    data_dir,
+    coords,
+    dim_values,
+    expression_summary_fmg_vals_fn: Callable[[List[tuple]], Dict[str, List]] = random_expression_summary_fmg_values,
+) -> str:
+    cube_dir = f"{data_dir}/expression_summary_fmg"
+    tiledb.Array.create(cube_dir, expression_summary_fmg_schema, overwrite=True)
+
+    with tiledb.open(cube_dir, mode="w") as cube:
+        logical_attr_values = expression_summary_fmg_vals_fn(coords)
+        assert all(
```

Comment:
Is there a reason this assertion would fail?

Reply:
no idea, just copied what was written for `create_expression_summary_cube`.

Reply:
That's a lot of repeated code. You can generalize the original `create_expression_summary_cube` to handle both cases and avoid the copy-paste.

Reply:
As I pointed out in another comment, I don't think generalizing repeated code into a function that can handle both cases is necessarily better if it makes function input overly verbose. There are a lot of schema-related variables ("_fmg") that would need to be parametrized as input arguments.